content/blog/2018-03-23-hydroscoper.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: "hydroscoper"
+title: ὕδωρ + σκοπῶ
+package_version: 1.0.0
+authors:
+  - name: Konstantinos Vantas
+    url: https://kvantas.github.io/
+date: 2018-03-23
+categories: blog
+topicid: 666666666
+tags:
+  - r
+  - community
+  - software
+  - review
+  - package
+  - data-access
+  - hydrology
+  - Greece
+  - hydroscope
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+> Hydrology is a concept to unify statistics, data analysis and numerical models in order to understand and analyze the endless circulation of water between the earth and its atmosphere.
+
+That's a lot alike Data Science, isn't it? Hydrologic Processes evolve in space and time, are extreme complex and we may never comprehend them. For this reason Hydrologists use models where their inputs and outputs are measurable variables: climatic and hydrologic data, land uses, vegetation coverage, soil type etc.  
+
+Not so many years ago, I used to struggle for these data: travelling 500 km away searching in underground storages, requesting from peers and institutes or even use sophistries to get them. Nowadays, hydrologic data come from a variety of sources such as satellites, online weather stations, national climatic databases, etc. But problems never end: these data have a variety of types and formats, errors, noise and a big amount of missing values. 
```

Comment:
"use sophistries" - I thought I knew what it meant but had to look it up. Made me smile. I wonder if it would help others get it more quickly if you added some extra meaning in parentheses...or maybe that's too much.